### ISSUE TITLE
A no-bot future

### ISSUE THREAD

**Issue by peeveswozere** [finndev/PokeBuddy#1031]
I apologize for letting me misuse this section to post a personal tought.

I may attract some hate but I want to share it with you fellas, short version: I'd be happy if every bot account get wrecked.

I remember first days of playing pogo, it was unbelievable, never seen this much life around and this number of new faces, awesome, trust me!
Everyone excited for a lvl 400/500 something, my first vaporeon lvl 700 who was a one man army of a pokemon. There was a pokestop somehow isolated with some benches, I put a lure there and oh god I met so many new pals you won't believe me!

We made a group out of nowhere, organized gym raids, hunts while going from a pub to another, simply awesome.

Then botters come up, gyarados 2200 into gyms while the average was 1000 for a decent player, people started losing interest and leaving the game, others started botting to keep up spreading this disease all over. The bot account I made is unavailable at the moment and I took off the old bot-free account, lvl 21 with a vaporeon 1700 and an exxecutor 1600, started playing again with it, it was amazing.

I'd love to see every bot account erased but care for avery word I say, EVERY bot, not some of those enough savy to stop earlier, it will make the game enjoyable again, give him the taste he loss.

Some used the "i live in a rural area!" excuse for botting but we know it's untrue, use Nox for that, bot is another thing.

Let me be clear, I botted too, i got every catchable pokemon including region locked ones, havent touched any gym and wouldn't touch them later too, I evolved and cared for that account but wouldn't be better if everyone of our creations get deleted and no one have ever the possibility to do them again? Just gps spoof, no automatization on catching and walking and farming stops.

In the end, I just wish I could get my game back the way it's inteded to get played. If even a single account get thru the so called wave, i'll continue with bots, but i'd really wish no one escape the net.

Sorry for the wall of text but I just want to target those complaining the ban telling them to remember what brought them into this game and what it used to be.

Have fun bros!

**Comment by AlexCtasy** [comment]
Well dude I dont know where you live maybe u from Brazil I guess, as i see your avatar photo I think u are brazilian, i dont know if there is a safe place to play pógo in street. But here in Argentina. forget to play pogo in street cause u back home without cellphone, really i tell you that.. With no hack Is great play i know it.. IS fair mode on, but in a good place to play and live, well I cannot play outside, cause a lot off people here, back home without cell, is not safe.. So i prefer to play just with fake GPS, i really dont worry for bots, but fake gps i prefer play with that, to came back to home without my iPhone.

Btw, Niantic dont have way to ban people as perma ban, I dont know why some people can join account, i was banned with that new mode of softban and just wait 12 hours or 1hour and i got unbanned, of course was easy softban days ago when u just spin 40 times pokestops and all is fine. But well.. Here we go.

**Comment by peeveswozere** [comment]
I said GPS spoof must stay, bot must go, you haven't to roam the streets if
you don't want to. I'm from South Italy by the way!

Il 13/ago/2016 09:11 AM, "AlexCtasy" <notifications@github.com> ha scritto:

**Comment by AlexCtasy** [comment]
Yeah I agree with you, if you think clearly, fake GPS is advantage too, Because people that dont use fake gps need to use cars, or go to specific area to get some pokemons, and a person with fake gps just use the joystick from home and get all pokemon that want.. Btw play fair is play with the app offical with no thirdy party programs.. Of course pogo was not created to play in a country like Argentina or other that people get scammed in street if they are using phone to play.

**Comment by peeveswozere** [comment]
GPS Spoof is an advantage, indoubtably, but let's think it this way, you've to catch a hundred magikarp for a gyarados, it will take you days with spoof even in a spawn zone, with bot? Less than 2 hours and you got yourself 400+ candies. And also, fair player can close the gap, if you bot there is no way, to demonstrate what, I may ask? Not your hard work, no gain, nothing, why even botting? I told you why, 90% of botters do that cause 10% aren't able to play vaguely fair and there is no way to fight a botter without another bot.

Dangerous street sound like an excuse to me, spoof is enough. We bot cause others bot. If every cheat account get deleted everyone wins, i'd like to hear more feedback on this from this community

**Comment by Enviouse** [comment]
I just thought I would leave my thoughts as well people are using ur hard work to make money off use u need to do somthing about it it's a free bot but if u go on eBay u can pay ppl to use it for

**Comment by peeveswozere** [comment]
On g2play you can buy a lvl 25 account for 20 euros, they farmed them

**Comment by Rab89** [comment]
i agree to the extent its a misuse of the whole aspect of the game, Niantic saw a big picture of everyone going pokemon hunting, working as a team and making new friends in the process, but my reason for botting was to say a big F#%K you to those lucky enough to be driven around all day and loot pokestops collect a huge amount of very good pokemon and rub it in everyones face, all i can say is, tables have now turned, if i get banned which tbh is most probable  i will go down knowing i had a good time, 
and i'd also like to remind you of what a guy posted around a week ago basically thanking the dev because his son wanted to be part of the pokemon go scene but was physically disabled and felt left out, Pokebuddy had  put a smile on the little boys face, if i didnt have hundreds of posts to go through i would have quoted him.

**Comment by osiriselite** [comment]
I like gps spoofing i lost my family in a bus accident only me and my brother got by im handy i have no legs and one arm my brother is in a coma i use it since i cant really be on my own unless my nurse helps me when shes around. Not all of us have the luxury to go out and meet fellow pogos. But i have met incredible people when i do get a chance to get out. To me spoofing is my best way to enjoy the game.

**Comment by peeveswozere** [comment]
We are misunderstanding again. I remember that guy's post, but can you figure out a guy having fun watching the bot collecting pokemons? I'm not taking off the possibility to spoof, he cant move from house, i get it, AlexTasy cant get out too and i get it, but everyone is fine if spoof is ok.

The osiriselite answer just pop out, this was EXACTLY what  i was talking about, he's having fun without the bot, the bot is just a hearthless piece of cold metal that have fun for you, spoofing is the only way, to me, to enjoy the game at home and avoiding ruining everyone's game

**Comment by AlexCtasy** [comment]
@peeveswozere Well I can get out my house cause here is not safe to play with an iphone in the street, well osiris problem is a thing that niantic do not think about that when they create the game right? to think about people that cannot move because was involved in accidents. Btw i hope one day they understand those people and dont think to ban people that use spoof.

**Comment by osiriselite** [comment]
Yes they dont take that into consideration at all. And yet they still act like victims ive spent countless amount of money on coins just to be able to keep up with the rest so ill admit i tried the bot lvld up two lvls and the errors/problems people are having happened so i just stopped while i could.

**Comment by AlexCtasy** [comment]
@osiriselite U have ur main account fine right? Or you get banned?

**Comment by peeveswozere** [comment]
Mind that atm spoofs like Nox works amazingly

**Comment by osiriselite** [comment]
Yes its still good i just resorted back to my gps spoofing no soft ban or permaban

**Comment by osiriselite** [comment]
Nox?

**Comment by peeveswozere** [comment]
Here you go buddy: http://en.bignox.com/

Troubleshoot: http://en.bignox.com/blog/?p=445

**Comment by osiriselite** [comment]
Thank ill check it out

**Comment by AlexCtasy** [comment]
Is safe to use NOX right now? No probably to get baanned?

**Comment by peeveswozere** [comment]
I'm using it with 0 problems, also some friends of mine

**Comment by lordofthereef** [comment]
Adding stops and gym "everyone" would be a big step in getting people to be happy with the game. Right now if you're not in a big city the game sucks. Or the experience is greatly reduced from those who do live in one.

Even with NOX people are trying to turn this game into something it is not. The point is to go out and explore, not sit in front of a computer screen and farm. I am not judging. I have botted and used nox, but I am fully aware that the way I chose to play the game is not as intended.

**Comment by peeveswozere** [comment]
I live in a rural area, no stops, no gyms, this is my only way, there is a motivation for a nox usage, bot usage have no justification

**Comment by lordofthereef** [comment]
You can justify it however you want, my friend. People justify botting too. Just look st the threads where people claim they have physical insbilities to play games. 

At the end of the day, I can't, with a straight face, say I believe some third party programs are ok while tigers are not when they are ALL against terms of service. All I say to anyone is use st your own risk and don't use third party software in an account you're too afraid to lose...
